stdlib.ispc

```diff
@@ -111,15 +111,9 @@ __declspec(safe, cost0) static inline uniform int __taskCount() { return __task_
 ///////////////////////////////////////////////////////////////////////////
 // Low level primitives
 
-__declspec(safe, cost0) static inline float16 float16bits(unsigned int16 a) { return __halfbits_varying_int16(a); }
+__declspec(safe, cost0) static inline float16 halfbits(unsigned int16 a) { return __halfbits_varying_int16(a); }
```

Comment:
Renamed for consistency.

Reply:
I don't think this is good idea. Current naming is after type names: `double`, `float`.

I also though that we should have a single `floatbits` function with overloaded parameters of type `int64`, `int32`, and `int16` - this way it's easier do meta-programming, as the name of function doesn't change. But I have doubts about "safety", as it's very easy to pass `int16` as `int32`.